lib/plugins/aws/invoke.js

```diff
@@ -70,6 +78,13 @@ class AwsInvoke {
       Payload: Buffer.from(JSON.stringify(this.options.data || {})),
     };
 
+    if (this.options.context) {
+      if (typeof this.options.context === 'object') {
+        this.options.context = JSON.stringify(this.options.context);
```

Comment:
Let's not override `context` on `this.options` (it's introduces side effect for eventual external consumers of the option)